google/cloud/bigtable/table.h

```diff
@@ -38,6 +38,16 @@ namespace google {
 namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
+/// The branch taken by a Table::CheckAndMutateRow operation.
+enum class MutationBranch {
+  kPredicateNotMatched,
+  ///< The predicate provided to CheckAndMutateRow did not match and the
```

Comment:
Q: Do we like the "after the fact" comments?  I couldn't see any others.

Reply:
Thank you for asking. Really, because I picked this habit a long time ago: the formatting for `enum` comments was all wonky with "before the fact" comments, but it is fine now. Fixed.